src/configure/configure.ts

```diff
@@ -185,12 +187,21 @@ class Orchestrator {
             if (!this.inputs.azureSession) {
                 this.inputs.azureSession = getAzureSession();
             }
+
+            // Right click scenario not supported for Azure and local repo
+            if (this.isResourceAlreadySelected()) {
+                if (this.inputs.sourceRepository.repositoryProvider === RepositoryProvider.AzureRepos || this.isLocalRepo) {
+                    throw Error(Messages.GithubRepoRequired);
+                } else if (!extensionVariables.enableGitHubWorkflow) {
+                    // For github repo, we create a github pipeline
+                    extensionVariables.enableGitHubWorkflow = true;
+                    vscode.window.showInformationMessage("Creating github pipeline.");
```

Comment:
This should be "Since this is right click scenario, the setting "<setting-name>" will be ignored"